src/ontology/mondo-edit.obo

```diff
@@ -237945,10 +237943,10 @@ xref: GARD:3521 {source="MONDO:GARD"}
 xref: MEDGEN:1676827 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MESH:C537445 {source="MONDO:equivalentTo"}
 xref: OMIM:309580 {source="Orphanet:73220", source="Orphanet:93971/ntbt", source="Orphanet:93971", source="MONDO:equivalentTo", source="Orphanet:93973", source="Orphanet:73220/e", source="Orphanet:93974", source="GARD:0000081", source="Orphanet:93974/ntbt", source="Orphanet:93973/ntbt"}
-xref: Orphanet:73220 {source="MONDO:equivalentObsolete", source="MONDO:preferredExternal", source="OMIM:309580"}
-xref: Orphanet:93970 {source="OMIM:309580"}
+xref: Orphanet:73220 {source="MONDO:equivalentObsolete", source="OMIM:309580"}
+xref: Orphanet:93970 {source="MONDO:equivalentObsolete", source="OMIM:309580"}
```

Comment:
this term is not obsolete in orphanet: https://www.orpha.net/en/disease/detail/93970?name=93970&mode=orpha

Reply:
I did not see the message that "This entity has been excluded from the Orphanet nomenclature of rare diseases and moved to [Alpha-thalassemia-X-linked intellectual disability syndrome](https://www.orpha.net/en/disease/detail/847)"
My apologies. What you did, using "MONDO:equivalentObsolete" was correct. Thank you!

Reply:
Ok, no further change needed here.